ui/src/components/setting/SettingBilling.vue

```diff
@@ -27,44 +27,63 @@
             <v-spacer />
 
             <v-col
-              v-if="isOwner && !active"
+              v-if="isOwner && inactive"
               md="auto"
               class="ml-auto"
             >
               <PaymentMethod
                 type-operation="subscription"
                 data-test="subscriptionPaymentMethod-component"
-                @update="getSubscriptionInfo()"
               />
             </v-col>
           </v-row>
 
           <div class="mt-6 pl-4 pr-4">
-            <div v-if="isOwner && !active">
-              <p>
-                Plan: <b> Free </b>
-              </p>
-
-              <p>
-                Description: you can only add 3 devices.
+            <div
+              v-if="pending && !retrialExceeded"
+              data-test="pending-retrial"
+            >
+              <p class="ma-4">
+                You have a pending request, please wait a while ...
               </p>
             </div>
-
-            <div v-else-if="isOwner && active">
-              <p>
-                Plan: <b> Premium usage </b>
+            <div
```

Comment:
You can add a blank line, because it separates one block from another and is better for previewing.